File: Python Code/Daily_Flash_Code_0/Week10/25Mar/56-DailyFlash_Solutions/25-MarSolution_Three/Python/prog4.py
num = 1
for i in range(7):
    if(i < 4):
        for j in range(i + 1):
            print(num, end = "")
            if(j != 3):
                num+=1
    else:
        for j in range(7 - i):
            num-=1
            print(num, end = "")
    print()

File: Python Code/Daily_Flash_Code_0/Week10/25Mar/56-DailyFlash_Solutions/25-MarSolution_Three/C++/prog4.cpp
#include<iostream>

int main() {
	int num = 1;
	for(int i = 0; i < 7; i++) {
		if(i < 4) {
			for(int j = 0; j < i + 1; j++) {
				std::cout << num;
				if(j != 3)
					num++;	
			} 
		} else {
			for(int j = 0; j < 7 - i; j++) {
				num--;
				std::cout << num;
			}
		}
		std::cout << "\n";
	}
}
